integration_tests/tests/integration/in_cluster.rs

```diff
@@ -32,8 +38,185 @@ struct InClusterJanusPair {
+    ///      "token": "Krx-CLfdWo1ULAfsxhr0rA"
+    ///    }
+    async fn new_in_cloud(vdaf: VdafInstance, query_type: QueryType) -> Self {
+        let (
+            divviup_api_url,
+            divviup_api_token,
+            divviup_account_id,
+            leader_aggregator_id,
+            helper_aggregator_id,
+            collector_credential_id,
+            collector_credential,
+        ) = match (
+            env::var("JANUS_E2E_DIVVIUP_API_URL"),
+            env::var("JANUS_E2E_DIVVIUP_API_TOKEN"),
+            env::var("JANUS_E2E_DIVVIUP_ACCOUNT_ID"),
+            env::var("JANUS_E2E_LEADER_AGGREGATOR_ID"),
+            env::var("JANUS_E2E_HELPER_AGGREGATOR_ID"),
+            env::var("JANUS_E2E_COLLECTOR_CREDENTIAL_ID"),
+            env::var("JANUS_E2E_COLLECTOR_CREDENTIAL_JSON"),
+        ) {
+            (
+                Ok(divviup_api_url),
+                Ok(divviup_api_token),
+                Ok(divviup_account_id),
+                Ok(leader_aggregator_id),
+                Ok(helper_aggregator_id),
+                Ok(collector_credential_id),
+                Ok(collector_credential_json),
+            ) => (
+                divviup_api_url.parse().unwrap(),
+                divviup_api_token,
+                Uuid::parse_str(&divviup_account_id).unwrap(),
+                Uuid::parse_str(&leader_aggregator_id).unwrap(),
+                Uuid::parse_str(&helper_aggregator_id).unwrap(),
+                Uuid::parse_str(&collector_credential_id).unwrap(),
+                serde_json::from_str::<PrivateCollectorCredential>(&collector_credential_json)
+                    .unwrap(),
+            ),
+            _ => panic!("missing or invalid environment variables"),
+        };
+
+        let mut divviup_api = DivviupClient::new(
+            divviup_api_token,
+            Client::new(RustlsConfig::<ClientConfig>::default()).with_default_pool(),
+        );
+        divviup_api.set_url(divviup_api_url);
+
+        let aggregators = divviup_api.aggregators(divviup_account_id).await.unwrap();
+        let leader_aggregator_dap_url = aggregators
+            .iter()
+            .find(|a| a.id == leader_aggregator_id)
+            .map(|a| &a.dap_url)
+            .unwrap();
+        let helper_aggregator_dap_url = aggregators
+            .iter()
+            .find(|a| a.id == helper_aggregator_id)
+            .map(|a| &a.dap_url)
+            .unwrap();
+
+        let (mut task_parameters, task_builder) = test_task_builder_remote(
+            vdaf,
+            query_type,
+            leader_aggregator_dap_url,
+            helper_aggregator_dap_url,
+            Duration::from_secs(30),
+            Duration::from_secs(600),
+        );
+        let task = task_builder.with_min_batch_size(100).build();
+        task_parameters.min_batch_size = 100;
+
+        let provision_task_request = NewTask {
+            name: format!("Integration test task {}", Uuid::new_v4()),
```

Comment:
Putting a UUID in the task's name seems like it could be confusing, when the task will get another UUID assigned to it in divviup-api's database. I suggest we insert a date and time here instead. (same for the other NewTask occurrence)

Reply:
Good idea -- I think name collisions don't matter, and timestamp will be sufficient to let us tell one from the other.